### PR TITLE
fix(chat): fix autoscrolling for next chat and chat regeneration (Issues #501, #500)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
- Issue: https://github.com/epam/ai-dial-chat/issues/501

### CODE REVIEW

**Review by mikitabut — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-801.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-801.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/Chat.tsx [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { memo, useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { useTranslation } from 'next-i18next';
 
 import { clearStateForMessages } from '@/src/utils/app/clear-messages-state';
-import { throttle } from '@/src/utils/data/throttle';
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Chat.tsx:6]: Please, remove this file

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-801.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**